src/abstractdataframe/io.jl

```diff
@@ -134,26 +134,8 @@ end
 #
 ##############################################################################
 
-if VERSION ≥ v"0.7.0-DEV.4059"
-    function latex_char_escape(char::Char)
-        if char == '\\'
-            return "\\textbackslash{}"
-        elseif char == '~'
-            return "\\textasciitilde{}"
-        else
-            return string('\\', char)
-        end
-    end
-else
-    function latex_char_escape(char::AbstractString)
-        if char == "\\"
-            return "\\textbackslash{}"
-        elseif char == "~"
-            return "\\textasciitilde{}"
-        else
-            return string("\\", char)
-        end
-    end
+function latex_char_escape(char::Char)
+    char == '\\' ? "\\textbackslash{}" : char == '~' ? "\\textasciitilde{}" : string('\\', char)
```

Comment:
The old version seemed more readable to me, even if it's longer.

Reply:
I reverted to the earlier version.